src/Symfony/Component/HttpKernel/DependencyInjection/Extension.php

```diff
@@ -33,12 +34,24 @@ public function getClassesToCompile()
     }
 
     /**
+     * Gets the annotated classes to cache.
+     *
+     * @return array An array of classes
+     */
+    public function getAnnotatedClassesToCompile()
+    {
+        return $this->annotatedClasses;
+    }
+
+    /**
      * Adds classes to the class cache.
      *
-     * @param array $classes An array of classes
+     * @param array $classes          An array of class patterns
+     * @param array $annotatedClasses An array of class patterns
      */
-    public function addClassesToCompile(array $classes)
+    public function addClassesToCompile(array $classes, array $annotatedClasses = array())
```

Comment:
rather than adding a second argument, I suggest adding a new method. It will make the code more readable